### PR TITLE
Add dates to all content.

### PR DESCRIPTION
**Pull request by johnharris85**
This PR adds a `date` frontmatter key to all articles that don't currently have one. It also adds a `lastmod` frontmatter key to _all_ articles. We may or may not use the `lastmod` in the future (probably will) and this just preserves it now as this PR will touch all those files and ruin the built in git `lastmod` functionality. I've also set the global config to prefer the frontmatter `lastmod` over the git one (the default was the other way around).

Had a quick scan around the site and doesn't look like anything's broken so.....

Signed-off-by: John Harris <joharris@vmware.com>

### CODE REVIEW

File: content/blog/code-to-containers.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
 ---
-title: "From Commit to Container"
-featured: false
-description: >
-    Building Containers from your code.
 date: 2020-05-29
-topics:
-- Containers
+description: 'Building Containers from your code.
```
Comment (BrianMMcClain) [inline content/blog/code-to-containers.md:3]: Is there any way to rerun the script without adding these multiple newlines to the description? Seems a bit odd

**Review by BrianMMcClain — COMMENTED** [review]
@johnharris85 the other thing I noticed is that this seems to change the order of the items on the left-hand nav for guides. For some sections this isn't a big deal, but I know the order on, say, the Kubernetes section was pretty specific.

File: content/blog/code-to-containers.md (hunk 1 above)
Comment (johnharris85) [reply content/blog/code-to-containers.md:3]: So, before they were multiline / block strings in YAML using `>` (not sure why?) I guess this is how the YAML library translated them when writing back to the file, but.........I checked the pages and it doesn't actually seem to display / show them. So functionally no effect (afaics), but I agree it's weird.

Comment (johnharris85) [reply content/blog/code-to-containers.md:3]: I guess I could re-post-process them all and remove all \n from description fields? Idk, lmk?

**Review by BrianMMcClain — APPROVED** [review]

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**